### ISSUE TITLE
Question for Xeditable

### ISSUE THREAD

**Issue by kvdogan** [pivotal-energy-solutions/django-datatable-view#168]
Hi @tiliv, I really would like to know if there is any possibility to have a validation for xeditable inputs, or any workaround making it possible to use make_xeditable with helper function (i.e 'through',  or any custom functions same as we utilize with helper function). thanks in advance for the answer.

**Comment by colindjk** [comment]
I'd also like to know this, have you heard anything back from the dev team?

As of now I plan on adding those features from scratch somehow.

**Comment by kvdogan** [comment]
No, unfortunately not yet. I have added model level validation as a workaround.

**Comment by tiliv** [comment]
We'll have to look at that for 1.1 and tie it to the view with a form class.  We do use a form class internally, but it's a pretty dynamic affair.  There should be something we can set up to make the situation easier to handle.
